boxoffice/views/admin_discount.py

```diff
@@ -9,13 +10,20 @@
 from utils import xhr_only, date_time_format
 
 
+# Move this to settings maybe
+PAGE_SIZE = 6
```

Comment:
I was thinking if we should make it a global config. so we just need to change it in one place. defining it inside the view also works for now.